README.md

```diff
@@ -150,11 +215,105 @@ Keep your `customers` database up to date with billing changes by
 - Listen to `Events on your account`
 - Select `customer.subscription.created` and `customer.subscription.deleted`
 
-### Deno Deploy
+#### Customer Portal Branding
+
+[Set up your branding on Stripe](https://dashboard.stripe.com/settings/branding),
+as the user will be taken to Stripe's checkout page when they upgrade.
+
+### Deployment Options
+
+#### Deno Deploy
+
+These steps show you how to deploy your SaaS app close to your users at the edge
+with [Deno Deploy](https://deno.com/deploy).
+
+1. Create a GitHub repository for your SaaSKit project.
+
+2. Sign into [Deno Deploy](https://dash.deno.com) with your GitHub account.
+
+3. Select your GitHub organization or user, repository, and branch
+
+4. Select "Automatic" deployment mode and `main.ts` as the entry point
+
+5. Click "Link", which will start the deployment.
+
+6. Once the deployment is complete, click on "Settings" and add the production
+   environmental variables, then hit "Save"
+
+You should be able to visit your newly deployed SaaS.
+
+#### Deno Deploy via GitHub Action
+
+You can also choose to deploy to Deno Deploy via a
+[GitHub Action](https://github.com/features/actions), which offers more
+flexibility. For instance, with the GitHub Action, you could:
+
+- Add a build step
+- Run `deno lint` to lint your code
+- Run `deno test` to run automated unit tests
+
+1. Create
+   [a new, empty project from the Deno Deploy dashboard](https://dash.deno.com/new).
+   Set a name for your project.
+
+2. Add the GitHub Action.
+
+[GitHub Actions](https://docs.github.com/en/actions) are configured using a
+`.yml` file placed in the `.github/workflows` folder of your repo. Here's an
+example `.yml` file to deploy to Deno Deploy. Be sure to update the
+`YOUR_DENO_DEPLOY_PROJECT_NAME` with one that you've set in Deno Deploy.
+
+```yml
+# Deploy this project to Deno Deploy
+name: Deploy
+on: [push]
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write  # Needed for auth with Deno Deploy
+      contents: read  # Needed to clone the repository
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+
+      - name: Install Deno
+        uses: denoland/setup-deno@main
+        # If you need to install a specific Deno version	
+        # with:
+        #   deno-version: 1.32.4
+
+## You would put your building, linting, testing and other CI/CD steps here
+
+## FInally, deploy 
+      - name: Upload to Deno Deploy
+        uses: denoland/deployctl@v1
+        with:
+          project: YOUR_DENO_DEPLOY_PROJECT_NAME
+          entrypoint: main.ts
+          # root: dist
+          import-map: import_map.json
+          exclude: .git/** .gitignore .vscode/** .github/** README.md .env .example.env
+```
+
+3. Commit and push your code to GitHub. This should trigger the GitHub Action.
+   When the action successfully completes, your app should be available on Deno
+   Deploy.
```

Comment:
What if we instead pointed to https://github.com/denoland/deployctl/blob/main/action/README.md and just mentioned making `main.ts` the entry point?

Reply:
lets do both!

Reply:
👍🏾